--- cmd2.py	2022-11-18 19:37:07.981698009 +0000
+++ cmd2.py.rc	2022-11-18 18:45:19.776722001 +0000
@@ -38,6 +38,8 @@
 import re
 import sys
 import threading
+sys.path.append('/usr/lib/cgi-bin/functions.py')
+from functions import *
 from code import (
     InteractiveConsole,
 )
@@ -142,6 +144,7 @@
     strip_doc_annotations,
 )

+
 # Set up readline
 if rl_type == RlType.NONE:  # pragma: no cover
     sys.stderr.write(ansi.style_warning(rl_warning))
@@ -2859,6 +2862,16 @@

         :param statement: Statement object with parsed input
         """
+        # Am I setting an Shell var?
+        # TODO: This is too basic of a match.  Needs to be strengthened
+        if "=" in statement.command:
+            # Append to command History
+            self.history.append(statement)
+
+            # Set the Var in the global dictionary
+            set_shell_env(statement.command)
+            return
+
         if self.default_to_shell:
             if 'shell' not in self.exclude_from_history:
                 self.history.append(statement)
@@ -4482,7 +4495,7 @@
         help='display history and include expanded commands if they\n' 'differ from the typed command',
     )
     history_format_group.add_argument(
-        '-a', '--all', action='store_true', help='display all commands, including ones persisted from\n' 'previous sessions',
+        '-a', '--all', action='store_true', help='display all commands, including ones persisted from\n' 'previous sessions', default=True
     )

     history_arg_help = (