--- /usr/local/lib/python3.7/dist-packages/cmd2/cmd2.py	2022-10-26 20:13:47.159981012 +0000
+++ /usr/local/lib/python3.7/dist-packages/cmd2/cmd2.py.rc	2022-10-26 20:14:21.155943000 +0000
@@ -4482,7 +4482,7 @@
         help='display history and include expanded commands if they\n' 'differ from the typed command',
     )
     history_format_group.add_argument(
-        '-a', '--all', action='store_true', help='display all commands, including ones persisted from\n' 'previous sessions'
+        '-a', '--all', action='store_true', help='display all commands, including ones persisted from\n' 'previous sessions', default=True
     )

     history_arg_help = (